screens/UserTimeCardsFilterForm.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { Field, reduxForm, FieldArray } from 'redux-form'
+import { ScrollView, Text, View, TouchableOpacity, Image, ActivityIndicator, FlatList, Modal } from 'react-native'
+import { DismissKeyboard } from '../components/DismissKeyboard'
+import BackBtn from '../components/BackBtn'
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome'
+import Icon from 'react-native-vector-icons/Ionicons'
+import { getMonthName } from '../actions'
+import images from '../assets/images'
+import styles from '../styles'
+import { LocaleContext } from '../locales/LocaleContext'
+import DropDown from '../components/DropDown'
+
+class UserTimeCardsFilterForm extends React.Component {
```

Comment:
It looks like UserTimeCardsFilterForm and StaffTimeCardFilterForm are almost the same. Can you refactor it so they are reusable component that you can insert into staff time card screen and user time cards screen? Thanks.